docker/httpd/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM alpine
+RUN mkdir -p /var/www/html/
+COPY . /var/www/html/
+WORKDIR /var/www/html/
+
+RUN apk add --no-cache yarn && \
+    yarn install && \
+    yarn run encore production
+
+FROM httpd:2.4-alpine
```

Comment:
Apache? :wink: Consider using nginx ...

Reply:
I did consider it. However, the whole platform (apart from Stepup) uses Apache. I see no reason to change it.